torchbenchmark/models/pytorch_unet/__init__.py

```diff
@@ -70,6 +70,10 @@ def train(self):
             grad_scaler.step(optimizer)
             grad_scaler.update()
 
+    def jit_callback(self):
+        assert self.jit, "Calling JIT callback without specifying the JIT option."
+        self.model = torch.jit.freeze(torch.jit.script(self.model.eval()), preserved_attrs=["n_classes"])
```

Comment:
I think this only applies when `self.test == 'eval'`. If running the train test, `torch.jit.freeze` doesn't apply.

```suggestion
        if self.test == 'eval':
            self.model = torch.jit.optimize_for_inference(torch.jit.freeze(torch.jit.script(self.model.eval()), preserved_attrs=["n_classes"]))
```

Reply:
OK